tests/acceptance/repo/build-list-routes-test.js

```diff
@@ -127,32 +92,60 @@ test('build history shows, more can be loaded, and a created build gets added an
     assert.ok(page.builds(2).errored, 'expected the third build to have errored');
 
     assert.ok(page.showMoreButton.exists, 'expected the Show More button to exist');
-
-    assert.equal(page.builds(3).name, 'rarely-used', 'expected the old default branch to show');
```

Comment:
I'm not sure, maybe Curtis removed it to make the test pass when working on it and then he forgot to put it back? I'll revert this and see how it goes.